### DISCUSSION TITLE
Possibility to allow single objects per specific unique field

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2983] [Archive]
I like single objects collection but I was wondering if is possible to create smth like this per specific unique field? For example, we can have a user id field and based on that create a single object?

**Comment by benhaynes** [comment]
I... don't know if I understand. Isn't that the Item Detail page? eg: `/member/uuid-pk-here`

**Reply by botzill** [reply]
Right now single object collection allows 1 object in general. But would be nice to have 1 object per `user_created`.
